### ISSUE TITLE
Golang emailpassword change password

### ISSUE THREAD

**Issue by trivialkettle** [supertokens/docs#704]
Hi,
I spotted an error at the golang code for changePasswordAPI: https://supertokens.com/docs/emailpassword/common-customizations/change-password

```
    // call signin to check that the input is correct
    isPasswordValid, err := emailpassword.SignIn(sessionContainer.GetTenantId(), userInfo.Email, requestBody.OldPassword)
    if err != nil {
        // TODO: Handle error
        return
    }

    if isPasswordValid.OK != nil {
        // TODO: Handle error
        return
    }
```

This should be

```
    // call signin to check that the input is correct
    isPasswordValid, err := emailpassword.SignIn(sessionContainer.GetTenantId(), userInfo.Email, requestBody.OldPassword)
    if err != nil {
        // TODO: Handle error
        return
    }

    if isPasswordValid.WrongCredentialsError != nil {
        // TODO: Handle error
        return
    }
```

Note the `if isPasswordValid.WrongCredentialsError != nil {` check

**Comment by rishabhpoddar** [comment]
Thanks for this! We will fix this in the next docs update (will be out in a few days).

**Comment by trivialkettle** [comment]
Hi, I checked you commit that references this issue and did not see a fix for this issue, though you fixed my other issue #705 .

**Comment by rishabhpoddar** [comment]
Oh yea, i was fixing both together along with some other stuff, and messed up the commit links. This is fixed: https://github.com/supertokens/docs/commit/ed1279865bd8fb9a114cf829277bd12039e9b026
